llvm/test/CodeGen/AMDGPU/default_amdhsa_code_object_version.ll

```diff
@@ -0,0 +1,7 @@
+; RUN: llc -mtriple=amdgcn-amd-amdhsa %s -o - | FileCheck %s
+
+; CHECK: .amdhsa_code_object_version 6
```

Comment:
It seems we have no other tests checking this field, which seems hard to believe. We probably should have one that tests it with explicit flags etc.

Reply:
We already have multiple tests that have explicit module flag and checks for the match.

Reply:
`git grep "\.amdhsa_code_object_version" llvm/test/CodeGen/AMDGPU/` does not find them

Reply:
ah, they check the IR match Lol. I will add one later then.